chia/full_node/full_node.py

```diff
@@ -202,6 +202,7 @@ def sql_trace_callback(req: str):
                             # this is a new DB file. Make it v2
                             async with self.db_wrapper.write_db() as w_conn:
                                 await set_db_version_async(w_conn, 2)
+                                self.db_wrapper.db_version = 2
```

Comment:
It might be safer to re-create `self.db_wrapper` instead of poking at its internals. Open to suggestions.

Reply:
oh, I see. I think this is reasonable.
hopefully we can remove this code in a year or so, when we only support v2